src/main.css

```diff
@@ -2299,4 +2299,16 @@
   .project-card code {
     background: #333 !important;
   }
+  /* fix gh circlebadge bg e.g. https://github.com/marketplace/dependabot-preview */
+  .CircleBadge--github::after {
+    background-color: var(--color-icon-primary);
+  }
+  .CircleBadge-icon[alt*="workflow logo"] {
+    filter: invert(.95) brightness(1.5) grayscale(1);
+    mix-blend-mode: lighten;
+  }
+  /* fix GH circlebadge https://github.com/login/oauth/authorize?client_id=c602a8bd54b1e774f864&scope=repo */
+  .CircleBadge[style*="background-color: #fff; overflow"] {
+    background-color: var(--color-bg-tertiary);
+  }
```

Comment:
Hi @the-j0k3r.
I copied and pasted your code, is that okay?

Reply:
Its fine, however check that it fixes the examples given with GHD.

Also those css vars need to be assigned values overriding the existing, these CSS vars need to be added to vars.css

Reply:
What color do you prefer?

Reply:
> What color do you prefer?

I dont prefer any color, I would perhaps use already defined colors based on GHD palette, something that works when user selects other colors other than what --ghd-bg-color gives you, ya good luck, this style is insane since it doesnt add all available CSS vars and overrides them, so the colors will never fit in